cala-ledger/src/tx_template/entity.rs

```diff
@@ -0,0 +1,364 @@
+use derive_builder::Builder;
```

Comment:
did you commit the file above `cala-ledger/src/tx_template/core.rs` by accident?